src/test/macromanager_test.cpp

```diff
@@ -0,0 +1,85 @@
+#include "recording/macromanager.h"
+
+#include <gtest/gtest.h>
+
+#include <QDebug>
+
+#include "mixxxtest.h"
+#include "signalpathtest.h"
+
+TEST(MacrosTest, CreateMacro) {
+    auto macro = new Macro();
+    ASSERT_EQ(macro->m_length, 0);
+}
+
+TEST(MacroManagerTest, ClaimRecording) {
+    auto mgr = MacroManager();
```

Comment:
I'd prefer one of these:

```suggestion
    MacroManager macroManager;
```
or:
```suggestion
    auto pMacroManager = new MacroManager();
    // ...
    delete pMacroManager;
```

But I think it the best approach would be to use a `m_pMacroManager` member variable and initialize it in the constructor of MacroManagerTest class. That way we don't need to repeat this code all the time.

Reply:
These tests don't use the test fixture, that's why it has to be repeated. I'd rather keep the use of test fixtures to a minimum, especially if it only saves a single line.
But I will incorporate your suggestions :)